src/test/kotlin/dev/reimer/tex/gradle/plugin/TexTests.kt

```diff
@@ -22,16 +22,16 @@ class TexTests : StringSpec({
         SimpleLatexBiberIncludeTest
     )
     configs.forEach { test ->
-        log.debug("Test to be executed: $test from ${test.directory}")
+        log.debug("Test to be executed: {} from {}", test, test.directory)
```

Comment:
Why not use the standard string interpolation syntax here?
```suggestion
        log.debug("Test to be executed: $test from ${test.directory}")
```

Reply:
JetBrains suggests this behavior, see https://www.jetbrains.com.cn/en-us/help/inspectopedia/LoggingStringTemplateAsArgument.html

Reply:
As minimal efficiency gains like with this trick do not really matter for this library, I'd like to stick to the more readable (old) syntax here.